src/commands/ExportCommand.js

```diff
@@ -0,0 +1,68 @@
+import Command from "../Command";
+import ChildProcessUtilities from "../ChildProcessUtilities";
+import FileSystemUtilities from "../FileSystemUtilities";
+import async from "async";
+import path from "path";
+import userHome from "user-home";
+
+export default class ExportCommand extends Command {
+  initialize(callback) {
+    this.dry = this.flags.dryRun;
+
+    this.pkg = this.input[0];
+    this.to = this.input[1] || userHome;
+
+    if (this.dry) {
+      this.logger.info("The following directories would be moved");
+    }
+
+    callback(null, true);
+  }
+
+  execute(callback) {
+    if (!this.to) {
+      // we actually know that this is an error earlier, during initialization,
+      // but calling the initialization errback with an error doesn't halt
+      // execution and print a nice error like the execute errback.
+      return callback(new Error("no directory found to export to; please provide a second argument"));
+    }
+
+    if (this.pkg) {
+      const pkg = this.packageGraph.get(this.pkg);
+
+      if (!pkg) {
+        if (this.to) {
+          return callback(`no such package to export ${this.pkg}`);
+        } else {
+          // they're not exporting a single package to their home directory,
+          // they're exporting everything to a provided directory!
+          this.to = this.pkg;
+          this.pkg = undefined; // http://jsperf.com/delete-vs-undefined-vs-null/16
```

Comment:
I'm a little confused by the logic here, actually.  Looks like the outer if/else branch we're in here has already locked us into exporting a single `pkg`, and `this.pkg` actually isn't even used after this?